src/components/tests/appbar.test.tsx

```diff
@@ -9,39 +9,69 @@ const mount = createMount();
 describe('AppBar', () => {
   let toggleShowOmniPanel: jest.Mock;
   let showSettings: jest.Mock;
+  let showTour: jest.Mock;
 
   beforeEach(() => {
     toggleShowOmniPanel = jest.fn();
     showSettings = jest.fn();
+    showTour = jest.fn();
   });
 
   test('renders correctly', () => {
     const root = mount(
-      <AppBar toggleShowOmniPanel={toggleShowOmniPanel} showSettings={showSettings} />,
+      <AppBar
+        toggleShowOmniPanel={toggleShowOmniPanel}
+        showSettings={showSettings}
+        showTour={showTour}
+      />,
     );
     expect(root).toMatchSnapshot();
```

Comment:
Can you try to mount the component with `shallow` because it's generating a kind of unreadable snapshots with the props of material UI. The idea of the snapshots is to generate a readable file where besides of the comparison that the test tooling does, the programmer can look at the code that the component it's generating.

HTH

Reply:
Oh okay let me test this out